app/src/ui/create-branch/create-branch-dialog.tsx

```diff
@@ -348,12 +351,13 @@ export class CreateBranch extends React.Component<
   ) {
     if (defaultBranch === null || defaultBranch.name === currentBranchName) {
       return (
-        <p>
+        <div>
           Your new branch will be based on your currently checked out branch (
           <Ref>{currentBranchName}</Ref>
           ). <Ref>{currentBranchName}</Ref> is the {defaultBranchLink} for your
           repository.
-        </p>
+          {this.renderForkLink()}
```

Comment:
I think we can be a little more concise in these "non-selection" instances by conditionally adding prefix to the existing informative text of "As determined by your `<link>` for behavior settings `</link>`, y"our new branch will be based on your currently checked out branch.

Reply:
I have changed it to a suffix instead. Does that work?